src/pages/Subscribe/Subscribe.js

```diff
@@ -1,7 +1,362 @@
+  const [address, setAddress] = useState('');
+  const PRICE = 39000;
+
+  const result = PRICE * index;
+
+  const navigete = useNavigate();
+
+  const goToBack = () => {
+    navigete(-1);
+  };
+
+  const onChangeName = e => {
+    setName(e.target.value);
+  };
+  const onChangeAddress = e => {
+    setAddress(e.target.value);
+  };
+
+  const onIndex = e => {
+    setIndex(e.target.value);
+  };
+
+  const onClickNaverBtn = e => {
+    alert('네이버페이로 결제하시겠습니까?');
+  };
+
+  const onClickKakaoBtn = e => {
+    alert('카카오페이로 결제하시겠습니까?');
+  };
+
+  const onClickCardBtn = e => {
+    alert('신용카드로 결제하시겠습니까?');
+  };
+
+  useEffect(() => {
+    window.scroll(0, 0);
+  }, [pathname]);
+
+  let token = localStorage.getItem('token') || '';
+  return (
+    <div>
+      <Backbutton onClick={goToBack}>뒤로가기</Backbutton>
+      <SubscribeTitle>매월 몇 개의 담아박스가 필요하세요?</SubscribeTitle>
+      <SubscribeTitleBox>
+        <SubscribeTitleImage
+          src="https://cdn-icons-png.flaticon.com/512/679/679720.png"
+          alt="담아박스 이미지"
+        />
+        <RigntTextWapper>
+          <p>담아박스</p>
+          <SelectBox>
+            <SelectBoxIcon>x</SelectBoxIcon>
+            <SelectBoxList value={index} onChange={onIndex}>
+              <option value="1">1</option>
+              <option value="2">2</option>
+              <option value="3">3</option>
+              <option value="4">4</option>
+              <option value="5">5</option>
+            </SelectBoxList>
+          </SelectBox>
+          <RigntTextSecondText>
+            매월 <span>{result.toLocaleString()}</span>원
+          </RigntTextSecondText>
+          <RigntTextDetailText>
+            매월 셋째주 월요일마다 자동결제돼요!
+          </RigntTextDetailText>
+        </RigntTextWapper>
+      </SubscribeTitleBox>
+      <AddressBoxWapper>
+        <AddressPlusBox>
+          <AddressPlusBoxTitle>구독 배송지를 선택해주세요</AddressPlusBoxTitle>
+          <AddressPlusBoxButton>배송지 추가</AddressPlusBoxButton>
+        </AddressPlusBox>
+        <AddressInfoBox>
+          <AddressInfo>
+            <AddressInfoInput
+              onChange={onChangeName}
+              type="text"
+              placeholder="이름"
+            />
+            <AddressInfoInput type="text" placeholder="전화번호" />
+            <AddressInfoInput
+              onChange={onChangeAddress}
+              type="text"
+              placeholder="주소"
+            />
+          </AddressInfo>
+          <AddressInfoActionBox>
+            <p>
+              선택
+              <input type="checkBox" />
+            </p>
+            <AddressInfoAditButton>수정</AddressInfoAditButton>
+          </AddressInfoActionBox>
+        </AddressInfoBox>
+      </AddressBoxWapper>
+      <PayBox>
+        <PayTitle>결제방법</PayTitle>
+        <PayNaverButton onClick={onClickNaverBtn}>네이버페이</PayNaverButton>
+        <PayKakaoButton onClick={onClickKakaoBtn}>카카오페이</PayKakaoButton>
+        <PayCardButton onClick={onClickCardBtn}>신용카드</PayCardButton>
```

Comment:
3가지 모두 alert띄우는 용도로 쓰는거라, 한가지 함수로 해결할 수 있겠습니다.
```suggestion
        <PayNaverButton onClick={puchaseNotice}>네이버페이</PayNaverButton>
        <PayKakaoButton onClick={puchaseNotice}>카카오페이</PayKakaoButton>
        <PayCardButton onClick={puchaseNotice}>신용카드</PayCardButton>
```